### ISSUE TITLE
Qt5 support

### ISSUE THREAD

**Issue by francisco-dlp** [hyperspy/hyperspyUI#143]
Using the development version of pyside and traitsui HyperSpy (actually hyperspy_gui_traitsui) supports Qt5, however HyperSpyUI fails like this:

```
Traceback (most recent call last):
  File "/home/fjd29/anaconda3/bin/hyperspyui", line 11, in <module>
    sys.exit(main())
  File "/home/fjd29/anaconda3/lib/python3.6/site-packages/hyperspyui/__main__.py", line 75, in main
    from python_qt_binding import QtGui, QtCore, QT_BINDING
  File "/home/fjd29/anaconda3/lib/python3.6/site-packages/python_qt_binding/__init__.py", line 55, in <module>
    from .binding_helper import loadUi, QT_BINDING, QT_BINDING_MODULES, QT_BINDING_VERSION  # @UnusedImport
  File "/home/fjd29/anaconda3/lib/python3.6/site-packages/python_qt_binding/binding_helper.py", line 267, in <module>
    getattr(sys, 'SELECT_QT_BINDING_ORDER', None),
  File "/home/fjd29/anaconda3/lib/python3.6/site-packages/python_qt_binding/binding_helper.py", line 95, in _select_qt_binding
    raise ImportError("Could not find Qt binding (looked for: %s):\n%s" % (', '.join(["'%s'" % b for b in binding_order]), '\n'.join(error_msgs)))
ImportError: Could not find Qt binding (looked for: 'pyqt', 'pyside'):
  ImportError for 'pyqt': No module named 'PyQt4'
  ImportError for 'pyside': No module named 'PySide'
```

**Comment by vidartf** [comment]
Probably best to switch to using `qtpy`. Should hopefully not be too big of a switch.

**Comment by vidartf** [comment]
Closed by #150 .
